components/Achievements.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
```

Comment:
did not touch the component since we're getting the data from outside no need for any function calling in here just a simple change to the percentage so it's calculated correctly in the progress bar